packages/http-client-python/eng/scripts/ci/regenerate.ts

```diff
@@ -153,10 +153,17 @@ const EMITTER_OPTIONS: Record<string, Record<string, string> | Record<string, st
     namespace: "authentication.http.custom",
     "package-pprint-name": "Authentication Http Custom",
   },
-  "authentication/union": {
-    "package-name": "authentication-union",
-    namespace: "authentication.union",
-  },
+  "authentication/union": [
+    {
+      "package-name": "authentication-union",
+      namespace: "authentication.union",
+    },
+    {
+      "package-name": "setuppy-authentication-union",
+      namespace: "setuppy.authentication.union",
+      "keep-setup-py": "true",
```

Comment:
You could apply this change to https://github.com/Azure/autorest.python/blob/2f594b37becb5be429708af73193dfc1062dac6a/packages/typespec-python/scripts/eng/regenerate.ts, too.

Reply:
added